packages/ui/src/components/bible-reader.tsx

```diff
@@ -87,6 +90,23 @@ function Root({
     onChange: onVersionChange,
   });
 
+  const [currentFontSize, setCurrentFontSize] = useState(() => {
+    if (typeof window === 'undefined') return fontSize;
+    const size = localStorage.getItem('font-size');
+    return size ? parseInt(size) : fontSize;
+  });
+
+  const [currentFontFamily, setCurrentFontFamily] = useState(() => {
+    if (typeof window === 'undefined') return fontFamily;
+    return localStorage.getItem('font-family') || fontFamily;
+  });
+
+  useEffect(() => {
```

Comment:
This is minor, but we could split this into two different `useEffect()` hooks. If we leave it as it is, we'll be setting both font size and family in local storage every time that either value changes. 
This is one unnecessary call to localStorage.